tools/src/main/java/org/bitcoinj/tools/WatchMempool.java

```diff
@@ -33,12 +33,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.common.collect.ImmutableList;
-
 public class WatchMempool {
     private static final Logger log = LoggerFactory.getLogger(WatchMempool.class);
     private static final NetworkParameters PARAMS = MainNetParams.get();
-    private static final ImmutableList<Transaction> NO_DEPS = ImmutableList.of();
```

Comment:
@msgilligan why replace `ImmutableList` with `emptyList()`? Just curious.

Reply:
We've been migrating away from the Guava `ImmutableList` and related classes.  `Collections.emptyList()` is the best available replacement in JDK 8 for `ImmutableList.of()`.